src/components/EventSchedule/EventScheduleList.jsx

```diff
@@ -1,48 +1,48 @@
-import React, { Component } from "react";
+import React, { useEffect } from "react";
 import PropTypes from "prop-types";
-import EventSchedule from "./EventScheduleItem";
+import "./EventSchedule.scss";
+import SpinnerWrapper from "../../containers/SpinnerContainer/SpinnerContainer";
+import NoResult from "../shared/NoResult/NoResult";
+import EventCard from "../Landing/EventCard/EventCard";
+import { EVENT_NO_RESULT } from "../../constants/eventConstants";
 
-class EventSchedulesList extends Component {
-  renderItems = arr =>
-    arr.map(item => (
-      <EventSchedule
-        key={item.id}
-        item={item}
-        current_user={this.props.currentUser}
-      />
-    ));
-
-  render() {
-    const items = this.renderItems(this.props.dataList);
-    const { dataList } = this.props;
-    return (
-      <>
-        <div className="row">
-          {dataList.length > 0 ? (
-            items
-          ) : (
-            <div id="notfound" className="w-100">
-              <div className="notfound">
-                <div className="notfound-404">
-                  <div className="h1">No Results</div>
-                </div>
-              </div>
-            </div>
-          )}
-        </div>
-      </>
-    );
-  }
-}
+const EventSchedulesList = ({ events, getEvents, isDataFetched }) => {
+  useEffect(() => {
+    getEvents();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+  return (
+    <div className="container">
+      <SpinnerWrapper showContent={isDataFetched}>
+        {events.length > 0 ? (
```

Comment:
Can we make just one flag for this rendering?

Reply:
Yes